Assets/Scripts/DeathCollider.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
```

Comment:
No need to create a separate script to check whether player collided with DeathCollider or not. You can implement this thing in your player controller and check the same thing using GetComponent or Tag. Always avoid MonoBehaviour classes and code repetition.